conf/mainnet/node.sqlite.properties

```diff
@@ -14,6 +14,18 @@ DB.Optimize = on
 ## Info: MEMORY journal mode is not supported
 DB.SqliteJournalMode = WAL
 
+## Sqlite synchronous
+## https://www.sqlite.org/pragma.html#pragma_synchronous
+## Lowering this parameter can increase performance, but decrease reliability.
+## FULL (default) is the safest, but NORMAL should be also fine for running a local node.
+## Setting this to OFF could lead to database corruption on opertaing system crash or power loss!
+DB.SqliteSynchronous = FULL
```

Comment:
set to:

# DB.SqliteSynchronous = FULL

Reply:
Is it at least the right place to put it?